### PR TITLE
Some more symbols

### PR DESCRIPTION
**Pull request by dsslimshaddy**
wiki

### CODE REVIEW

File: lib/money/currency.ex [hunk 1]
```diff
@@ -87,21 +87,21 @@ defmodule Money.Currency do
...
     JMD: %{name: "Jamaican Dollar",                                        symbol: "J$",   exponent: 2},
-    JOD: %{name: "Jordanian Dinar",                                        symbol: " ",    exponent: 3},
+    JOD: %{name: "Jordanian Dinar",                                        symbol: "JOD",    exponent: 3},
     JPY: %{name: "Yen",                                                    symbol: "¥",    exponent: 0},
-    KES: %{name: "Kenyan Shilling",                                        symbol: " ",    exponent: 2},
+    KES: %{name: "Kenyan Shilling",                                        symbol: "KSh",    exponent: 2},
     KGS: %{name: "Som",                                                    symbol: "лв",   exponent: 2},
     KHR: %{name: "Riel",                                                   symbol: "៛",    exponent: 2},
-    KMF: %{name: "Comoro Franc",                                           symbol: " ",    exponent: 0},
+    KMF: %{name: "Comoro Franc",                                           symbol: "KMF",    exponent: 0},
```
Comment (Nitrino) [inline lib/money/currency.ex]: Symbol of Ethiopian Birr is "CF"

File: lib/money/currency.ex [hunk 2]
```diff
@@ -111,23 +111,23 @@ defmodule Money.Currency do
...
+    MMK: %{name: "Kyat",                                                   symbol: "K",    exponent: 2},
     MNT: %{name: "Tugrik",                                                 symbol: "₮",    exponent: 2},
-    MOP: %{name: "Pataca",                                                 symbol: " ",    exponent: 2},
-    MRO: %{name: "Ouguiya",                                                symbol: " ",    exponent: 2},
+    MOP: %{name: "Pataca",                                                 symbol: "MOP$",    exponent: 2},
+    MRO: %{name: "Ouguiya",                                                symbol: "UM",    exponent: 2},
     MUR: %{name: "Mauritius Rupee",                                        symbol: "₨",    exponent: 2},
-    MVR: %{name: "Rufiyaa",                                                symbol: " ",    exponent: 2},
-    MWK: %{name: "Kwacha",                                                 symbol: " ",    exponent: 2},
+    MVR: %{name: "Rufiyaa",                                                symbol: "ރ",    exponent: 2},
```
Comment (Nitrino) [inline lib/money/currency.ex]: I think it's better to use "Rf" or "MVR" symbol

File: lib/money/currency.ex [hunk 3]
```diff
@@ -136,7 +136,7 @@ defmodule Money.Currency do
     OMR: %{name: "Rial Omani",                                             symbol: "﷼",    exponent: 3},
     PAB: %{name: "Balboa US Dollar",                                       symbol: "B/.",  exponent: 2},
     PEN: %{name: "Nuevo Sol",                                              symbol: "S/.",  exponent: 2},
-    PGK: %{name: "Kina",                                                   symbol: " ",    exponent: 2},
+    PGK: %{name: "Kina",                                                   symbol: "PGK",    exponent: 2},
```
Comment (Nitrino) [inline lib/money/currency.ex]: Symbol of Ethiopian Birr is "K"

File: lib/money/currency.ex [hunk 4]
```diff
@@ -149,37 +149,37 @@ defmodule Money.Currency do
...
-    SLL: %{name: "Leone",                                                  symbol: " ",    exponent: 2},
+    SLL: %{name: "Leone",                                                  symbol: "Le",    exponent: 2},
     SOS: %{name: "Somali Shilling",                                        symbol: "S",    exponent: 2},
     SRD: %{name: "Surinam Dollar",                                         symbol: "$",    exponent: 2},
-    STD: %{name: "Dobra",                                                  symbol: " ",    exponent: 2},
+    STD: %{name: "Dobra",                                                  symbol: "Db",    exponent: 2},
     SVC: %{name: "El Salvador Colon US Dollar",                            symbol: "$",    exponent: 2},
     SYP: %{name: "Syrian Pound",                                           symbol: "£",    exponent: 2},
-    SZL: %{name: "Lilangeni",                                              symbol: " ",    exponent: 2},
+    SZL: %{name: "Lilangeni",                                              symbol: "L",    exponent: 2},
```
Comment (Nitrino) [inline lib/money/currency.ex]: Symbol "L" is used only for the singular. For the plural used "E" symbol

File: lib/money/currency.ex [hunk 5]
```diff
@@ -22,34 +22,34 @@ defmodule Money.Currency do
   """
 
   @currencies %{
-    AED: %{name: "UAE Dirham",                                             symbol: " ",    exponent: 2},
+    AED: %{name: "UAE Dirham",                                             symbol: "د.إ",    exponent: 2},
     AFN: %{name: "Afghani",                                                symbol: "؋",    exponent: 2},
     ALL: %{name: "Lek",                                                    symbol: "Lek",  exponent: 2},
-    AMD: %{name: "Armenian Dram",                                          symbol: " ",    exponent: 2},
+    AMD: %{name: "Armenian Dram",                                          symbol: "֏",    exponent: 2},
```
Comment (Nitrino) [inline lib/money/currency.ex]: I I think it's better to use "AMD" symbol

**Comment by Nitrino** [comment]
Thank you for the great work. 
We have several currencies with the same symbol, I think we should add disambiguate symbol

File: lib/money/currency.ex [hunk 6]
```diff
@@ -127,7 +127,7 @@ defmodule Money.Currency do
     MXV: %{name: "Mexican Peso Mexican Unidad de Inversion (UDI)",         symbol: "UDI",  exponent: 2},
     MYR: %{name: "Malaysian Ringgit",                                      symbol: "RM",   exponent: 2},
     MZN: %{name: "Metical",                                                symbol: "MT",   exponent: 2},
-    NAD: %{name: "Rand Namibia Dollar",                                    symbol: "$",    exponent: 2},
+    NAD: %{name: "Rand Namibia Dollar",                                    symbol: " ",    exponent: 2},
```
Comment (Nitrino) [inline lib/money/currency.ex]: I think it's better to fix tests, rather than change the currency symbol because of the tests

Comment (dsslimshaddy) [inline lib/money/currency.ex]: Yeah i thought that too Namibians wont be happy i removed their dollar

**Comment by Nitrino** [comment]
Thank you!

**Pull request merged**